includes/functions.php

```diff
@@ -300,7 +300,7 @@ function tsml_custom_post_types()
 			'show_ui' => true,
 			'has_archive' => $is_public,
 			'menu_icon' => 'dashicons-groups',
-			'rewrite' => ['slug' => $tsml_slug],
+			'rewrite' => ['slug' => $tsml_slug, 'with_front' => apply_filters('tslm_meeting_with_front', true)],
```

Comment:
we use PSR-12, looks like this might need some more indentation here. could you format this with PHP Intelephense?

Reply:
I think this is resolved but not 100% sure, please confirm.